components/director/internal/domain/formation/notificationbuilder.go

```diff
@@ -128,9 +126,8 @@ func (nb *NotificationBuilder) PrepareDetailsForConfigurationChangeNotificationG
 ) (*formationconstraintpkg.GenerateFormationAssignmentNotificationOperationDetails, error) {
 	details := &formationconstraintpkg.GenerateFormationAssignmentNotificationOperationDetails{
 		Operation:             operation,
-		FormationID:           formationID,
```

Comment:
The `FormationID` from the `formationconstraintpkg.GenerateFormationAssignmentNotificationOperationDetails` was used after when we built the webhook input(`FormationConfigurationChangeInput` and `ApplicationTenantMappingInput`). Now we have removed this field in favor of the formation structure that is also used at a later stage to populate the formationID property of the webhook input in case someone is using it in the Golang templates. It's an improvement in the code not to propagate different properties that we could use from already available variables.